buildcraft_resources/assets/buildcraftfactory/compat/buildcraft/guide/en_us/block/pump.xml

```diff
@@ -0,0 +1,15 @@
+<chapter name="tile.pumpBlock.name"/>
+<lore>
+With the collection of fluids by hand can often be quite tedious and dangerous at times, the need for an automated way to gather fluids was required.
+</lore>
+<no_lore>
+The pump is a way to automatically pick up any fluid sources found directly below it and any sources connected to it.
+</no_lore>
+<chapter name="Information"/>
+To gather fluids from the world, place a pump ontop of the highest fluid source block of the source you want to pump up and supply it with power.
+<recipes_usages stack="buildcraftfactory:pump"/>
+The fluids souces found below will be 'sucked' up from the top source downwards and will be auto-outputed into adjacent pipes/tanks.
+It will pump fluids in a 64 blocks radius from top to bottom and the outer edges of the connecting fluids inwards.
+<chapter name="Machine Mechanics"/>
+As with most machines, it requires power. The more power you supply it, the faster it will pump.
+Applying a pump with a redstone signal will stop the pump from pumping so be careful where you place your levers.
```

Comment:
thinking 'applying a redstone signal to the pump' might be better, this sounds like you are using redstone to apply the pump to the world